typing/printtyp.mli

```diff
@@ -88,6 +88,7 @@ module Conflicts: sig
   val reset: unit -> unit
 end
 
+val type_path: formatter -> Path.t -> unit
```

Comment:
I think it would be better to group the function with the `path` function above, with a short description of the difference between the two. Otherwise, it is not clear when one should use or the other. Even more so, when we don't want to use short-paths in case like
```ocaml
type best = ..
type a_worse__name = best
type a_worse__name  += A
```